Ports/lxt/package.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='lxt'
+version='1.3b'
+files="http://stahlke.org/dan/lxt/lxt-${version}.tar.gz lxt-${version}.tar.gz 7a3ab299a6d61a71b271fd13b847b5a1c22a5f95df78561a325c78d50b6a6bc7"
```

Comment:
The `files` property has had its syntax changed since the PR was opened.

```suggestion
files=(
    "http://stahlke.org/dan/lxt/lxt-${version}.tar.gz 7a3ab299a6d61a71b271fd13b847b5a1c22a5f95df78561a325c78d50b6a6bc7"
)
```

Reply:
@seal331 You still need to take care of this comment